kentikapi/internal/api_payloads/alerting.go

```diff
@@ -15,12 +16,12 @@ type CreateManualMitigationRequest struct {
 }
 
 type Alarm struct {
```

Comment:
Improvement: This should say: AlarmPayload, to make it consistent with the rest of the paylods naming